spec/services/api/v1.2/catalog/evaluate_order_process_spec.rb

```diff
@@ -0,0 +1,91 @@
+describe Api::V1x2::Catalog::EvaluateOrderProcess, :type => :service do
+  describe "#process" do
+    let(:order) { create(:order) }
+    let!(:order_item) { create(:order_item, :order => order, :portfolio_item => portfolio_item) }
+    let(:portfolio_item) { create(:portfolio_item, :portfolio => portfolio) }
+    let(:portfolio) { create(:portfolio) }
+
+    subject { described_class.new(order).process }
+
+    context "when there are no existing tags on the portfolio or portfolio_item" do
+      it "applies the process sequence of '1' to the order item" do
+        subject
+        expect(order.order_items.first.process_sequence).to eq(1)
+      end
+
+      it "applies the proccess scope of 'applicable' to the order item" do
+        subject
+        expect(order.order_items.first.process_scope).to eq("applicable")
+      end
+
+      it "does not add any other order items to the order" do
+        subject
+        expect(order.order_items.count).to eq(1)
+      end
+    end
+
+    context "when there is 1 existing tag on the portfolio_item" do
```

Comment:
Actually, looking through this, my `context` has the wrong description, because https://github.com/RedHatInsights/catalog-api/pull/786/files#diff-c74675ae9ec4d087ece4c738d736435aR56 is doing `portfolio.tag_add`.

So yeah, probably another test where we do `portfolio_item.tag_add` as well as when both of them have one.